prometheus-exporters/opensearch-logs-query-exporter/files/opensearch_logs_cronus_compliance.cfg

```diff
@@ -0,0 +1,84 @@
+[query_opensearch_logs_cronus_availability_check]
+# The DEFAULT settings can be overridden.
+QueryIntervalSecs = 3600
+QueryTimeoutSecs = 60
+QueryIndices = logs-ds
+QueryOnMissing = zero
+QueryJson = {
+  "size": 0,
+  "query": {
+    "bool": {
+      "must": [
+        {
+          "term": {
+            "resource.k8s.container.name.keyword": "cronus"
+          }
+        }
+      ],
+      "filter": [
+        {
+          "range": {
+            "@timestamp": {
+              "gte": "now-30d"
+            }
+          }
+        }
+      ]
+    }
+  },
+  "aggs": {
+    "daily_logs": {
+      "date_histogram": {
+        "field": "@timestamp",
+        "calendar_interval": "day",
+        "min_doc_count": 0,
+        "extended_bounds": {
+          "min": "now-30d",
+          "max": "now"
+        }
+      }
+    }
+  }
+}
+
+[query_opensearch_logs_cronus_compliance_violation]
```

Comment:
I think this name is not very clear to me. Couldn't a "compliance violation" be a lot of different things? So what would be a name for a future metric that also is a "compliance violation"?

I can see that for instance `opensearch_logs_cronus_compliance_violation_old_logs_count_value` gives a _bit_ more information for a user. But I would still argue that I am not certain what kind of compliance check is actually being monitored here. If it is a _particular_ comp-vio then I would name the metric in such a way that it is clear what it should be used for.